labgrid/resource/udev.py

```diff
@@ -708,3 +708,15 @@ def filter_match(self, device):
             return False
 
         return super().filter_match(device)
+
+@target_factory.reg_resource
+@attr.s(cmp=False)
+class USBResetPort(USBResource):
```

Comment:
Isn't this name too generic? Perhaps `BCUController`?

As BCU also supports power measurement, the same resource would also be used to export that functionality.

Reply:
Agree, I will change the port name to something more specific: BCUController or BCUPort